bundles/EcommerceFrameworkBundle/Model/ProductInterface.php

```diff
@@ -19,6 +19,11 @@
  */
 interface ProductInterface
 {
+    /**
+     * @return int
+     */
+    public function getId();
```

Comment:
Again a proper BC-break, which should be in a dedicated PR. Thanks 😊

Reply:
https://github.com/pimcore/pimcore/pull/7548